### PR TITLE
CI(eos_cli_config_gen): Restructured the eos_cli_config_gen molecule host vars part-3

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
## Change Summary

Restructured the eos_cli_config_gen molecule host vars part-3

## Related Issue(s)

Fixes #(https://github.com/aristanetworks/avd-internal/issues/171)

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Restructured the eos_cli_config_gen molecule host vars 

## How to test
Run eos_cli_config_gen molecule then run tox command to generate the coverage report.

molecule converge -s eos_cli_config_gen
tox -e coverage,report

| Module | Coverage | Comment |
| ------- | ---------- | ---------- |
| eos/agents.j2 | 100 | - |
| documentation/agents.j2 | 100 | - |
| eos/aliases.j2 | 100 | - |
| documentation/aliases.j2 | 100 | - |
| eos/application-traffic-recognition.j2 | 99 | 2 partial |
| documentation/application-traffic-recognition.j2 | 98 | 6 partial |
| eos/arp.j2 | 99 | 1 partial |
| documentation/arp.j2 | 99 | 1 partial |
| eos/as-path.j2 | 100 | - |
| documentation/as-path.j2 | 100 | - |
| eos/boot.j2 | 100 | - |
| documentation/boot.j2 | 98 | 1 partial |
| eos/class-maps.j2 | 98 | 1 partial |
| eos/class-maps-pbr.j2 | 100 | - |
| documentation/class-maps.j2 | 100 | - |
| eos/clock.j2 | 96 | 1 partial |
| documentation/clock.j2 | 97 | 1 partial |
| eos/config-comment.j2 | 100 | - |
| eos/cvx.j2 | 95 | 4 partial |
| documentation/cvx.j2 | 86 | 8 missing and 8 partial |
| eos/daemons.j2 | 97 | 1 partial |
| documentation/daemons.j2 | 100 | - |
| eos/dhcp-relay.j2 | 100 | - |
| documentation/dhcp-relay.j2 | 98 | 1 partial |
| documentation/dhcp-servers.j2 | 99 | 1 partial |
| eos/dhcp-servers.j2 | 100 | - |
| eos/domain-list.j2 | 100 | - |
| documentation/domain-list.j2 | 100 | - |
| eos/dot1x.j2 | 93 | 24 partial |
| documentation/dot1x.j2 | 96 | 8 partial |
| eos/dot1x_part2.j2 | 100 | - |
| eos/dps-interfaces.j2 | 97 | 3 partial |
| documentation/dps-interfaces.j2 | 100 | - |
| documentation/enable-password.j2 | 88 | 2 missing, 3 partial |
| eos/enable-password.j2 | 90 | 2 missing, 3 partial |
| eos/vrfs.j2 | 100 | - |
| documentation/vrfs.j2 | 100 | - |
### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/dps-interfaces.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/dps-interfaces.yml:1]: Not support on ATD lab devices

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/application-traffic-recognition.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/application-traffic-recognition.yml:1]: not supported on ATD lab devices

**Review by ClausHolbechArista — APPROVED** [review]

**Comment by ClausHolbechArista** [comment]
Compared coverage with devel and ensured no coverage is lost. A bit gained.

**Review by Shivani-gslab — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/dps-interfaces.yml (hunk 1 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/dps-interfaces.yml:1]: Not understood. This is a YAML file where `---` means new YAML document.

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/dps-interfaces.yml:1]: aah you are talking about the data models. NP. This PR is just for rearranging existing tests.

**Pull request merged**